spec/requests/provider_suggestions_spec.rb

```diff
@@ -17,36 +17,32 @@
       expect(JSON.parse(response.body)).to eq("error" => "Bad request")
     end
   end
-  context "when provider suggestion ids valid" do
-    let(:provider_suggestions) {
-      stub_request(:get, "#{Settings.teacher_training_api.base_url}/api/v3/provider-suggestions?query=Str")
-        .to_return(
-          headers: { "Content-Type": "application/vnd.api+json; charset=utf-8" },
-          body: File.new("spec/fixtures/api_responses/provider-suggestions.json"),
-        )
-    }
+  context "when provider suggestion is valid" do
+    %w(Str Str%E2%80%99).each do |provider_query|
```

Comment:
I'd want some explanation for the encoded string in the codebase without having to git-blame it. I'd never guess why it was like this from reading the code.